src/Lime/Protocol/Client/Channel.ts

```diff
@@ -21,16 +22,23 @@ export interface SessionChannel extends SessionListener {
   sendSession(session: Session): void;
 }
 
-abstract class Channel implements MessageChannel, CommandChannel, NotificationChannel, SessionChannel {
+export interface ProcessCommand extends CommandListener {

```

Comment:
To keep the implementation consistent with other languages SDK, rename this interface to CommandProcessorChannel

Reply:
I mean  `CommandProcessor`